### PR TITLE
CeloTool simplify contexts

### PR DESCRIPTION
**Pull request by codyborn**
### Description

Simplify the context management. Instead of having to worry about the service that you're deploying to, we're just including the service name in the context label. This makes it straightforward if we want to deploy full nodes to the Oracle context for example.

### Other changes

Increase Komenci timeout from five seconds to seven seconds.

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: .env.rc1 [hunk 1]
```diff
@@ -13,18 +13,21 @@ CLUSTER_DOMAIN_NAME="celo-testnet"
...
+AZURE_ORACLE_EASTUS2_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
+AZURE_ORACLE_EASTUS2_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
+AZURE_ORACLE_EASTUS2_REGION_NAME=eus2
+AZURE_ORACLE_WESTUS_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
+AZURE_ORACLE_WESTUS_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
+AZURE_ORACLE_WESTUS_REGION_NAME=wus
+AZURE_ORACLE_WESTEUROPE_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
+AZURE_ORACLE_WESTEUROPE_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
+AZURE_ORACLE_WESTEUROPE_REGION_NAME=weu
 AZURE_SOUTHBR_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
```
Comment (tkporter) [inline .env.rc1]: Should these be prefixed with `AZURE_KOMENCI_SOUTHBR` now?

File: .env.rc1 [hunk 2]
```diff
@@ -13,18 +13,21 @@ CLUSTER_DOMAIN_NAME="celo-testnet"
...
 AZURE_SOUTHBR_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
 AZURE_SOUTHBR_REGION_NAME=br
+AZURE_SEA_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
```
Comment (tkporter) [inline .env.rc1]: `AZURE_KOMENCI_SEA`?

File: .env.rc1 [hunk 3]
```diff
@@ -72,93 +75,86 @@ ORACLE_UNUSED_ORACLE_ADDRESSES=0xB93Fe7906ea4221b3fbe23412D18Ab1B07FE2F71,0x8d25
 # A list of every valid context. Must start with one of: gcp,aws,azure
 # each context should have its own environment variables, generally of the form
 # <context in all caps and underscores>_*
-CONTEXTS=azure-westus,azure-westeurope,azure-eastus2,gcp-us-west1,gcp-us-east1,gcp-asia-east1,gcp-europe-west1,gcp-southamerica-east1,azure-southbr
+CONTEXTS=azure-oracle-westus,azure-oracle-westeurope,azure-oracle-eastus2,gcp-oracle-us-west1,gcp-oracle-us-east1,gcp-oracle-asia-east1,gcp-oracle-europe-west1,gcp-oracle-southamerica-east1,azure-komenci-southbr,azure-komenci-sea
```
Comment (tkporter) [inline .env.rc1]: can you change all the `gcp-oracle` ones to `gcp-forno`?

File: .env.alfajores [hunk 4]
```diff
@@ -10,13 +10,13 @@ CLUSTER_DOMAIN_NAME="celo-testnet"
 
 TESTNET_PROJECT_NAME="celo-testnet-production"
 
-AZURE_EASTUS_AZURE_SUBSCRIPTION_ID=97e2b592-255b-4f92-bce0-127257163c36
-AZURE_EASTUS_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
-AZURE_EASTUS_REGION_NAME=eus
+AZURE_KOMENCI_EASTUS_AZURE_SUBSCRIPTION_ID=97e2b592-255b-4f92-bce0-127257163c36
+AZURE_KOMENCI_EASTUS_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
+AZURE_KOMENCI_EASTUS_REGION_NAME=eus
```
Comment (tkporter) [inline .env.alfajores]: Should all the `_REGION_NAME` env vars be `_AZURE_REGION_NAME`?

File: packages/celotool/src/lib/env-utils.ts [hunk 5]
```diff
@@ -152,22 +152,15 @@ export enum DynamicEnvVar {
   AZURE_SUBSCRIPTION_ID = '{{ context }}_AZURE_SUBSCRIPTION_ID',
   AZURE_TENANT_ID = '{{ context }}_AZURE_TENANT_ID',
   KUBERNETES_CLUSTER_NAME = '{{ context }}_KUBERNETES_CLUSTER_NAME',
-  FORNO_KUBERNETES_CLUSTER_NAME = '{{ context }}_FORNO_KUBERNETES_CLUSTER_NAME',
-  FORNO_AZURE_KUBERNETES_RESOURCE_GROUP = '{{ context }}_FORNO_AZURE_KUBERNETES_RESOURCE_GROUP',
-  FULL_NODES_KUBERNETES_CLUSTER_NAME = '{{ context }}_FULL_NODES_KUBERNETES_CLUSTER_NAME',
-  FULL_NODES_AZURE_KUBERNETES_RESOURCE_GROUP = '{{ context }}_FULL_NODES_AZURE_KUBERNETES_RESOURCE_GROUP',
+  AZURE_KUBERNETES_RESOURCE_GROUP = '{{ context }}_AZURE_KUBERNETES_RESOURCE_GROUP',
```
Comment (tkporter) [inline packages/celotool/src/lib/env-utils.ts:153]: nit: could you move this up to keep alphabetical order?

**Review by tkporter — COMMENTED** [review]
Thanks!! A couple small things

File: .env.rc1 (hunk 1 above)
Comment (codyborn) [reply .env.rc1]: Yes, thanks for catching this.

File: .env.rc1 [hunk 6]
```diff
@@ -75,93 +75,86 @@ ORACLE_UNUSED_ORACLE_ADDRESSES=0xB93Fe7906ea4221b3fbe23412D18Ab1B07FE2F71,0x8d25
...
 # A list of domains to provision the SSL certificate for
 FORNO_DOMAINS=rc1-forno-k8s.celo-testnet.org.,forno.celo.org.
 # All clusters must be in this VPC
 FORNO_VPC_NETWORK_NAME=rc1-network
 
-AZURE_WESTUS_FORNO_AZURE_KUBERNETES_RESOURCE_GROUP=mainnet-oracles-westus2
-AZURE_WESTUS_FORNO_KUBERNETES_CLUSTER_NAME=mainnet-oracles-westus2v1
-AZURE_WESTEUROPE_FORNO_AZURE_KUBERNETES_RESOURCE_GROUP=mainnet-oracles-westeurope
-AZURE_WESTEUROPE_FORNO_KUBERNETES_CLUSTER_NAME=mainnet-oracles-westeurope
+AZURE_ORACLE_WESTUS_FORNO_AZURE_KUBERNETES_RESOURCE_GROUP=mainnet-oracles-westus2
```
Comment (tkporter) [inline .env.rc1]: I think this whole block (lines 88 - 91) can be removed

File: .env.rc1 [hunk 7]
```diff
@@ -75,93 +75,86 @@ ORACLE_UNUSED_ORACLE_ADDRESSES=0xB93Fe7906ea4221b3fbe23412D18Ab1B07FE2F71,0x8d25
...
-AZURE_WESTUS_FULL_NODES_COUNT=5
-AZURE_WESTUS_FULL_NODES_DISK_SIZE=100
+AZURE_ORACLE_WESTUS_ORACLE_ADDRESS_AZURE_KEY_VAULTS=0x0aee051be85ba9c7c1bc635fb76b52039341ab26:mainnet-oracle0:mainnet-oracles-westus2,0xd3405621f6cdcd95519a79d37f91c78e7c79cefa:mainnet-oracle1:mainnet-oracles-westus2,0xe037f31121f3a96c0cc49d0cf55b2f5d6deff19e:mainnet-oracle2:mainnet-oracles-westus2,0x12bad172b47287a754048f0d294221a499d1690f:mainnet-oracle3:mainnet-oracles-westus2,0xacad5b2913e21ccc073b80e431fec651cd8231c6:mainnet-oracle4:mainnet-oracles-westus2
+AZURE_ORACLE_WESTUS_FULL_NODES_COUNT=5
+AZURE_ORACLE_WESTUS_FULL_NODES_DISK_SIZE=100
 
-AZURE_WESTEUROPE_AZURE_KUBERNETES_RESOURCE_GROUP=mainnet-oracles-westeurope
-AZURE_WESTEUROPE_KUBERNETES_CLUSTER_NAME=mainnet-oracles-westeurope
+AZURE_ORACLE_WESTEUROPE_AZURE_KUBERNETES_RESOURCE_GROUP=mainnet-oracles-westeurope
```
Comment (tkporter) [inline .env.rc1]: Looks like these are duplicated on lines 97 and 98

**Review by tkporter — APPROVED** [review]
🙏 thanks so much!

**Pull request merged**